index.html

```diff
@@ -0,0 +1,537 @@
+	<link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@300&display=swap" rel="stylesheet">
+	<title>Smashing Magazine Clone</title>
+	<link rel="stylesheet" type="text/css" href="style/styles.css" />
+</head>
+
+<body>
+	<!-- /header -->
+	<header id="header" class="">
+		<nav class="navbar">
+			<img class="main-logo" src="images/logo--full.svg" alt="">
+			<div> <button class="navbar-button" type="button">Articles</button> </div>
+			<div> <button class="navbar-button" type="button">Guides</button> </div>
+			<div> <button class="navbar-button" type="button">Books</button> </div>
+			<div> <button class="navbar-button" type="button">Events</button> </div>
+			<div> <button class="navbar-button" type="button">Membership</button> </div>
+			<div> <button class="navbar-button" type="button">Jobs</button> </div>
+			<div> <button class="navbar-button" type="button">Newsletter</button> </div>
+			<div class="more-button">
+				<button style="padding-right: 0px;" class="buIcon" type="submit">More </button><img class="menu-cat-img"
+					src="images/menu-cat.svg" alt="">
+			</div>
+			<div class='formContainer'>
+				<form>
+					<div class='inputWhite'><i class="fa fa-search fa-lg" aria-hidden="true"></i><input
+							placeholder="search articles..." class="searchForm" type='text'></div>
+				</form>
+			</div>
+		</nav>
+	</header>
+	<main>
+		<h1 class="latestArticlesHighlight"><span>Don’t Miss These Latest <br> Articles</span></h1>
+		<div class="latest-articles font-balmasiq">
+			<div style="background: #555555;" class='profileCon'>
+				<div class='imgContainer'>
+					<div class="profileImageDiv"><img class='profileImg' src='images/kelvin-omereshone-200x200.jpg'
+							alt=''></div>
+					<div>
+						<p class="profileName">Kevin Omershone <em>wrote</em></p>
+					</div>
+
+				</div>
+				<h1>What Vitruvius Can Teach Us About Web Design <span class='inline'><i class="fa fa-envelope-o"
+							aria-hidden="true"></i> Comment</span>
+				</h1>
+				<div class='spanContainer'><span># React<sup> 45</sup> </span><span># Javascript<sup>
+							274</sup></span><span># Tools<sup> 229</sup></span></div>
+				<i class="fa fa-long-arrow-right fa-3x" aria-hidden="true"></i>
+			</div>
+			<div style="background: #757575;" class='profileCon'>
+				<div class='imgContainer'>
+					<div class="profileImageDiv"><img class='profileImg' src='images/carmine-zaccagnino-200x200.jpg'
+							alt=''></div>
+					<div>
+						<p class="profileName">Carmin Zaccagnino <em>wrote</em></p>
+					</div>
+				</div>
+				<h1>Solving Common Cross-Platform Issues When Working with Flutter 
+					<span class='inline'><i class="fa fa-envelope-o" aria-hidden="true"></i> Comment</span>
+				</h1>
+				
+				<div class='spanContainer'><span>Smashy Monthly Roundup<sup> 9</sup></span></div>
+				<i class="fa fa-long-arrow-right fa-3x" aria-hidden="true"></i>
+			</div>
+			<div style="background: #858585;" class='profileCon'>
```

Comment:
please remove all the inline styles from your HTML and put it in CSS files

Reply:
The changes are implemented.